src/assets/styles/var.sass

```diff
@@ -137,7 +137,7 @@ $app-color-action-list-item-inline-divider: #d0d7de7a
     border-radius: 1px
     position: relative
     float: left
-    border-color: $app-color-border-default
+    border-color: $app-color-border-default!important
```

Comment:
Please don't use `!important`,
I assume this was for the button hover state?
Could you please just add a `:hover` rule with needed styles

Reply:
Fixed